File: basis programmeren/codebase s/array/array.js
let programmeertalen = ["PHP", "C#", "HTML", "JavaScript", "Java", "C++", "C"];

programmeertalen.push("Python", "Swift");

programmeertalen.sort();

for (let taal of programmeertalen) {
  console.log(taal);
}

let aantalItems = programmeertalen.length;
console.log("Aantal items in de array: " + aantalItems);

File: basis programmeren/codebase s/array/array.py
programmeertalen = ["PHP", "C#", "HTML", "JavaScript", "Java", "C++", "C"]

programmeertalen.extend(["Python", "Swift"])

programmeertalen.sort()

for taal in programmeertalen:
    print(taal)
aantalItems = len(programmeertalen)
print("Aantal items in de array: " + str(aantalItems))
